service/confirm-reminder/src/index.ts

```diff
@@ -92,24 +98,32 @@ const job = schedule.scheduleJob('* * * * *', async () => {
 
 syncQueue(amqp_url, queueConfirm, async (action: ActionMessageV2 | EventMessageV2) => {
   if (action.schema === 'proca:action:2' && action.contact.dupeRank === 0) {
-
     console.log(`New confirm `, action.actionId);
-    try {
-      // ignore if we have it
-      const _payload =  await db.get('action-' + action.actionId);
-    } catch (_error) {
-      console.error('catch', _error);
-      const error = _error as LevelError;
-
-      if (error.notFound) {
-        await db.put<string, ActionMessageV2>('action-' + action.actionId, action, {});
-        const retry = { retry: changeDate(action.action.createdAt, 1, retryArray), attempts: 1 };
-        await db.put<string, RetryRecord>('retry-' + action.actionId, retry, {});
-
-        console.log(`Scheduled confirm reminder: ${action.actionId}`, action);
-      } else {
-        console.error(`Error checking if confirm scheduled in DB`, error);
-        throw error;
+
+    // Don't remind if action from the queue is too old
+    if (retryExpired(action.action.createdAt, maxPeriod)) {
```

Comment:
please avoid wrapping everything into ifs, but instead exit early on the opposite, ie:

```
   if (!retryExpired(action.action.createdAt, maxPeriod))
     return;
  // do the normal things if retry isn't expired
```

rationale: 
- avoid deeply nested code and loads of indentation makes it easier to read
- easier to reason, check for errors first and exit early, then the default function path is what it should do by default

Reply:
> please avoid wrapping everything into ifs, but instead exit early on the opposite, ie:
> 
> ```
>    if (!retryExpired(action.action.createdAt, maxPeriod))
>      return;
>   // do the normal things if retry isn't expired
> ```
> 
> rationale:
> 
>     * avoid deeply nested code and loads of indentation makes it easier to read
> 
>     * easier to reason, check for errors first and exit early, then the default function path is what it should do by default

@tttp check if ok now